shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -38,10 +38,10 @@
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
 import org.apache.accumulo.core.security.Authorizations;
+import org.apache.accumulo.core.spi.scan.DefaultScanInterpreter;
+import org.apache.accumulo.core.spi.scan.ScanInterpreter;
```

Comment:
I think I prefer the term `ScanFormatter` or something, since the job really is to format the results of a scan for display.

Reply:
The ScanInterpreter was not for formatting. It's job was to transform user inputs on the scan command.  For example if using the following command I think it would translate the row to binary before scanning.

```
scan -fi  org.apache.accumulo.core.util.interpret.HexScanInterpreter -r a7b9c0d1
```